sdc/tests/tests_perf/test_perf_series.py

```diff
@@ -217,11 +217,12 @@ def {func_name}(self):
     ('idxmax', '', [10 ** 9]),
     ('idxmin', '', [10 ** 9]),
     ('isna', '', [2 * 10 ** 7]),
-    ('max', '', [10 ** 9]),
+    ('max', '', [10 ** 8]),
+    ('max', 'skipna=False', [10 ** 8]),
```

Comment:
Shouldn't we also add parameters 'skipna=False' for `mean`, `median`, `idxmax`, `idxmin`, `var`, `cumsum` and so on?
@PokhodenkoSA

Reply:
I think we should check that it influence on scaling first. Then we should add this cases to show difference.